MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java

```diff
@@ -3,25 +3,35 @@
 import com.leapmotion.leap.Controller;
 import com.leapmotion.leap.Frame;
 import com.leapmotion.leap.Hand;
-import com.nautigsam.mineleapmod.helpers.LogHelper;
-import com.nautigsam.mineleapmod.lwjglVirtualInput.VirtualMouse;
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.gui.ScaledResolution;
 import net.minecraftforge.fml.client.FMLClientHandler;
+import net.minecraft.client.settings.GameSettings;
+
+import com.nautigsam.mineleapmod.helpers.LogHelper;
+import com.nautigsam.mineleapmod.lwjglVirtualInput.VirtualMouse;
+import com.nautigsam.mineleapmod.lwjglVirtualInput.VirtualKeyboard;
+import com.nautigsam.mineleapmod.helpers.McObfuscationHelper;
 
 public final class LeapMotionMouse {
 
+	public enum HandType {
+		RIGHT,
+		LEFT,
+	};
+
 	private static volatile LeapMotionMouse instance = null;
 
 	private Minecraft mc = FMLClientHandler.instance().getClient();
+	private GameSettings settings = Minecraft.getMinecraft().gameSettings;
```

Comment:
Could you modify this to match with the previous line? Newer Minecraft client doesn't know getMinecraft() method.